utils-macro/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "miden-node-utils-macro"
+version = "0.1.0"
+edition = "2021"
```

Comment:
Are we planning to publish this on crates.io? If not, I thin this is fine. But if we do plan to publish it, then we should use all our standard fields here.